packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2119,7 +2310,11 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
     if (_currentBottomSheet != null) {
       _currentBottomSheet._widget.animationController?.dispose();
     }
-    _floatingActionButtonMoveController.dispose();
+    _fabStatuses.forEach(
```

Comment:
Again you should use a for loop here (and just below here).

https://dart.dev/guides/language/effective-dart/usage#avoid-using-iterableforeach-with-a-function-literal

Reply:
(*See above*)